python/lsst/ts/electrometer/commander.py

```diff
@@ -0,0 +1,72 @@
+import asyncio
+import logging
+from lsst.ts import tcpip
+
+
+class TCPIPCommander:
+    """Implement communication with the electrometer.
+
+    Attributes
+    ----------
+    log
+    reader
+    writer
+    reply_terminator
+    command_terminator
+    lock
+    host
+    port
+    timeout
+    connected
+    """
+
+    def __init__(self) -> None:
+        self.log = logging.getLogger(__name__)
+        self.reader = None
+        self.writer = None
+        self.reply_terminator = b"\n"
+        self.command_terminator = "\r"
```

Comment:
`b"\r"` or `"\r"` here? This means we should express the "binary" or not.

Reply:
The command terminator is decoded because it is always generating a decoded command.